doc/src/manual/variables.md

```diff
@@ -136,7 +136,7 @@ conventions:
     camel case instead of underscores.
   * Names of `function`s and `macro`s are in lower case, without underscores.
   * Functions that write to their arguments have names that end in `!`. These are sometimes called
-    "mutating" or "in-place" functions because they are intended to produce changes in their arguments
+    "[mutating](https://discourse.julialang.org/t/understanding-mutating-functions-correctly/30651)" or "in-place" functions because they are intended to produce changes in their arguments
```

Comment:
I don't think we usually add links to discourse like this. If this information is missing from the manual it can be added and then we can add a proper reference to that section here.

Reply:
Hello,
Mutating key word details is present in the manual. 
When I was going through the document, I want to know about the key word mutating - if there was a referencing link then that would help. For the same I have added this link.
Also on the key word "Kronecker product" - I feel this content needs to be added in the manual.

Reply:
Also I feel we needs to provide link to REPL key word - which has information in the same document.
Key word backslashed need to be replaced with backslash.